securedrop/tests/test_i18n_tool.py

```diff
@@ -314,9 +315,13 @@ def r():
         ])
         assert 'l10n: updated nl' not in r()
         assert 'l10n: updated de_DE' not in r()
-        message = str(git('--no-pager', '-C', 'securedrop', 'show',
-                          _cwd=d, _encoding='utf-8'))
-        assert "Loïc" in message
+        if six.PY2:
```

Comment:
This could just use `six.string_type(git(...))`.

Reply:
`str.u` does not work for our test case and creates exception on `six` itself.

Reply:
`six.text_type` should work here